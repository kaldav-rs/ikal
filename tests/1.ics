BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
