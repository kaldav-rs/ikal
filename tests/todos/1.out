Ok(
    VTodo {
        dtstamp: Local(
            2007-03-13T13:34:32+01:00,
        ),
        uid: Text {
            params: {},
            text: "20070313T123432Z-456553@example.com",
        },
        class: Some(
            Confidential,
        ),
        completed: None,
        created: None,
        dtstart: None,
        geo: None,
        last_modified: None,
        location: None,
        organizer: None,
        percent_complete: None,
        priority: None,
        recurid: None,
        sequence: None,
        status: Some(
            NeedsAction,
        ),
        summary: Some(
            Text {
                params: {},
                text: "Submit Quebec Income Tax Return for 2006",
            },
        ),
        url: None,
        rrule: None,
        due: Some(
            Date(
                2007-05-01,
            ),
        ),
        duration: None,
        attach: [],
        attendee: [],
        categories: [
            Text {
                params: {},
                text: "FAMILY",
            },
            Text {
                params: {},
                text: "FINANCE",
            },
        ],
        comment: [],
        contact: [],
        exdate: [],
        rstatus: [],
        related_to: [],
        resources: [],
        rdate: [],
        x_prop: {},
        iana_prop: {},
    },
)
